mssql-ballerina/types.bal

```diff
@@ -0,0 +1,56 @@
+// import ballerina/jballerina.java;
+// import ballerina/sql;
+
+## Represents Point MsSQL Field
+#
+# + value - Value of parameter passed into the SQL statement
+public class PointValue {
+    public PointRecordType | string? value;
+
+    public function init(PointRecordType | string? value = ()) {
+        self.value =value;
+    }
+}
+
+## Represents LineString MsSQL Field
+#
+# + value - Value of parameter passed into the SQL statement
+public class LineStringValue {
+    public LineStringRecordType | string? value;
+
+    public function init(LineStringRecordType | string? value = ()) {
+        self.value =value;
+    }
+}
+
+# Represents Money MsSQL Field
+#
+# + value - Value of parameter passed into the SQL statement
+public class MoneyValue {
+    public decimal|float|string? value;
+    public isolated function init(decimal|float|string? value = ()) {
+        self.value = value;
+    }  
+}
+
+# Represents Smallmoney MsSQL Field
+#
+# + value - Value of parameter passed into the SQL statement
+public class SmallMoneyValue {
+    public decimal|float|string? value;
+    public isolated function init(decimal|float|string? value = ()) {
+        self.value = value;
+    }  
+}
+
+public type PointRecordType record {
```

Comment:
```suggestion
public type Point record {
```

Reply:
Do not suffix `RecordType`. Please check and correct all places

Reply:
Fixed with [`19b168a`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/19b168ad9bdf7ab04eee060816cf5209a2a80729)